### ISSUE TITLE
Is this still being worked on or updated?

### ISSUE THREAD

**Issue by Icedude907** [blu-dev/smash-visualizer#8]
Just wondering, I really enjoy using this tool - but I'm wondering if its been worked on without the updates being published here.
Thanks

**Comment by Feradile** [comment]
no, i dont think so, hes been working on a new project that visualizes stage collision

**Comment by Icedude907** [comment]
That's a pity. Does anyone know if the source code will ever come out?
